Tests/test_PDB.py

```diff
@@ -0,0 +1,88 @@
+# Copyright 2021 by Simon Duerr.  All rights reserved.
+# This code is part of the Biopython distribution and governed by its
+# license.  Please see the LICENSE file that should have been included
+# as part of this package.
+"""Tests for PDB module."""
+
+import gzip
+import sys
+import unittest
+import warnings
+
+from Bio import PDB
+
+from Bio.PDB import PDBParser, MMCIFParser
+
+
+class PDBTestBaseClass(unittest.TestCase):
+    """Base class for Bio.PDB unit tests."""
+
+    def setUp(self):
+        self.PDBParser = PDBParser()
```

Comment:
I'd rename these to `self.pdb_parser` and `self.mmcif_parser` instead of the uppercase variable names.

Reply:
Good point, in line with PEP8 conventions about classes vs variables and attributes.